src/dmd/dtoh.d

```diff
@@ -0,0 +1,1724 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1999-2019 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/dtohd, _dtoh.d)
+ * Documentation:  https://dlang.org/phobos/dmd_dtoh.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/dtoh.d
+ */
+module dmd.dtoh;
+
+import core.stdc.stdio;
+import core.stdc.string;
+import core.stdc.ctype;
+
+import dmd.astcodegen;
+import dmd.arraytypes;
+import dmd.globals;
+import dmd.identifier;
+import dmd.json;
+import dmd.mars;
+import dmd.root.array;
+import dmd.root.file;
+import dmd.root.filename;
+import dmd.root.rmem;
+import dmd.visitor;
+import dmd.tokens;
+
+import dmd.root.outbuffer;
+import dmd.utils;
+
+//debug = Debug_DtoH;
+
+private struct DMDType
+{
+    __gshared static Identifier c_long;
+    __gshared static Identifier c_ulong;
+    __gshared static Identifier c_longlong;
+    __gshared static Identifier c_ulonglong;
+    __gshared static Identifier c_long_double;
+    //version(BUILD_COMPILER)
+    //{
```

Comment:
?

Reply:
Thats a leftover refactor of mine from Iain's PR to make it a separate program and remove the DMD specific bits of the diff. https://github.com/dlang/dmd/pull/10801#discussion_r380643827